### PR TITLE
JavaScript in compiled mode MODMS-50

### PR DESCRIPTION
**Pull request by adamdickmeiss**
TODO: Dockerfile

### CODE REVIEW

**Comment by skoczko** [comment]
@adamdickmeiss Regarding Dockerfile see my comment on the ticket: https://issues.folio.org/browse/MODMS-50?focusedCommentId=131192&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-131192

File: pom.xml [hunk 1]
```diff
@@ -118,20 +115,24 @@
...
       <dependency>
         <groupId>org.graalvm.js</groupId>
         <artifactId>js</artifactId>
-        <version>22.1.0</version>
+	<version>${graalvm.version}</version>
+	<scope>runtime</scope>
       </dependency>
       <dependency>
         <groupId>org.graalvm.js</groupId>
         <artifactId>js-scriptengine</artifactId>
```
Comment (skoczko) [inline pom.xml:131]: Drop this dependency -- needed only if using the ScriptEngine API

File: server/pom.xml [hunk 2]
```diff
@@ -57,6 +58,10 @@
       <groupId>com.jayway.jsonpath</groupId>
       <artifactId>json-path</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.graalvm.sdk</groupId>
+      <artifactId>graal-sdk</artifactId>
```
Comment (skoczko) [inline server/pom.xml:63]: Transitive.

Comment (adamdickmeiss) [reply server/pom.xml:63]: OK removing.

File: pom.xml (hunk 1 above)
Comment (adamdickmeiss) [reply pom.xml:126]: Tests fail without it.

Comment (skoczko) [reply pom.xml:131]: Weird, I dropped it on. my branch and they are passing fine.

**Comment by adamdickmeiss** [comment]
Closing as work is done in #24

**Pull request closed without merging**